clang/test/Driver/sycl-offload.cpp

```diff
@@ -132,3 +132,9 @@
 // RUN:  %clangxx -### -Wl,-rpath,%S -fsycl -fintelfpga %t_empty.o %s 2>&1 \
 // RUN:    | FileCheck -check-prefix NO_DIR_CHECK %s
 // NO_DIR_CHECK-NOT: clang-offload-bundler: error: '{{.*}}': Is a directory
+
+/// Check that exception handline is disabled for device for Windows
```

Comment:
From a quick check, it looks like we don't have a similar test for Linux - is the behavior enforced there as well?

Reply:
It is not.  A separate PR can be put together to cover Linux.

Reply:
Got it, a separate PR sounds like a good plan. Thanks!